compiler/extccomp.nim

```diff
@@ -704,22 +704,6 @@ proc addExternalFileToCompile*(conf: ConfigRef; filename: AbsoluteFile) =
     flags: {CfileFlag.External})
   addExternalFileToCompile(conf, c)
 
-proc compileCFiles(conf: ConfigRef; list: CfileList, script: var Rope, cmds: var TStringSeq,
```

Comment:
why inline this proc?

Reply:
It's only used once and I found it confusing/misleading since it doesnt actually compile the C files.